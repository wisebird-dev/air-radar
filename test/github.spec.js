import React from 'react';
import {shallow} from 'enzyme';

import Github from '../src/components/github';

test('it works', () => {
	const wrapper = shallow(<Github/>);
	expect(wrapper).toMatchSnapshot();
});
