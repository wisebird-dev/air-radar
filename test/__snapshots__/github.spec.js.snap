// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it works 1`] = `
<StyledComponent
  forwardedComponent={
    Object {
      "$$typeof": Symbol(react.forward_ref),
      "attrs": Array [],
      "componentStyle": ComponentStyle {
        "componentId": "sc-bdVaJa",
        "isStatic": true,
        "rules": Array [
          "
  color: #ffffff;
  font-size: 14px;
  font-weight: 500;
  padding: 8px 10px;
  border-radius: 3px;
  transition: all .3s ease;
  background-color: #333;
  background-position: center;
  text-decoration: none;
  user-select: none;

  &:hover {
      background-color: #444;
  }
",
        ],
      },
      "displayName": "styled.a",
      "foldedComponentIds": Array [],
      "render": [Function],
      "styledComponentId": "sc-bdVaJa",
      "target": "a",
      "toString": [Function],
      "warnTooManyClasses": [Function],
      "withComponent": [Function],
    }
  }
  forwardedRef={null}
/>
`;
